src/main/webapp/authentication-app/src/main-components/SignInPage.js

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from "react";
+import {Link} from "@reach/router";
+
+const SignInPage = () => {
+  const [userPassword, setUserPassword] = useState('');
+  const [userEmail, setUserEmail] = useState('');
+  const [error, setError] = useState(null);
+
+  const signInHandler = (event,email,password) => {
+    event.preventDefault();
+  };
+
+  const onChangeHandler = (event) => {
+      const {name, value} = event.currentTarget;
+      {/* gets the . part of the address to check if its edu */}
+      const emailEnd = name.substring(name.length - 4, name.length);
```

Comment:
This should use a regex or at least a find - is name properly trimmed? What if they have spaces at the end?